### DISCUSSION TITLE
Oceanic Waypoint Coordinates not loading from Simbrief

### DISCUSSION THREAD

**Discussion by ow1962** [flybywiresim/aircraft#4523] [Help]
The oceanic waypoints give an "airway/wpt mismatch" on the mcdu and don't load when importing flight plan from simbrief. Tried to manually insert and I get "not in database".  All other waypoints load including oceanic entry and exit.  Using the latest experimental version. Any help?

**Comment by tracernz** [comment]
Which navdata are you using in your sim, and simbrief. What is your flightplan?

**Comment by ow1962** [comment]
Hi Michael:

Navigraph aircycle 2103 rev 2

I have attached the flight plan

when I checked my plan, saw that all the oceanic's were missing, I manually tried to enter into the mcdu

tried normal formats - 50N030W, 5000N03000W. and shortform  for example - all rejected as not in the database.

Thanks for the help

Andrew
________________________________

**Reply by tracernz** [reply]
Ahhhh. lat/lon format is not yet accepted, only named waypoints. Unfortunately your attached flightplan didn't come through.

**Comment by ow1962** [comment]
thanks

that explains it. oceanic crossing lanes are lat/long

Andrew
________________________________

**Reply by tracernz** [reply]
Stay tuned. We aim to support this format in future (already some of the groundwork is done).

**Comment by ow1962** [comment]
Hi Michael

I guess the PDF was too big .

Here is a text version just for your info

ICAO FLIGHT PLAN
----------------
FF LSASZQZX LFFFZQZX LFEEZQZX EGTTZQZX EGGXZQZX EISNZQZX CZQXZQZX
CZULZQZX
201335 CYULSBFP
(FPL-A20N-IS
-A20N/M-SDE3FGHIRWY/LB1
-LSGG1405
-N0443F380 DCT DIPIR UB37 DJL UH10 AMODO UM729 RESMI UN491 PIGOP
UN160 JSY UN502 MUREL UL739 LIZAD L739 MABUG N12 LULOX DCT
DOGAL/M075F370 NATA 53N050W/N0435F370 NATA RIKAL/N0440F380 N440A YRI
DCT DEBUS OMBRE8
-CYUL0747 CYOW
-PBN/A1B1C1D1O1S1 DOF/210420 REG/A20N EET/LFMM0000 LFFF0001 LFEE0004
EGTT0112 EGGX0133 EISN0141 EGGX0221 54N020W0247 CZQX0338 54N040W0426
53N050W0515 CZUL0605 PER/C RALT/EIKN BIKF CYYT RMK/TCAS)
-

oceanic crossing
OCEANIC CLEARANCE:
DOGAL 54N020W 54N030W 54N040W 53N050W RIKAL FL370 .75

________________________________
